scico/test/linop/test_radon_svmbir.py

```diff
@@ -40,6 +42,24 @@ def make_A(im, num_angles, num_channels):
     return A
 
 
+def cg_prox(f, v, λ):
+    # prox:
+    #   arg min  1/2 || x - v ||^2 + λ α || A x - y ||^2_W
+    #      x
+    #
+    # solution at:
+    #   (I + λ 2α A^T W A) x = v + λ 2α A^T W y
```

Comment:
Missing square root: should be A^T W^(1/2) y

Reply:
You sure? With the new definition of W I think it is only W, not W^(1/2)

`||y||_W^2 = y^T W y`

Reply:
Yes, you're right - the perils of late night mental algebra! This also needs to be corrected in  docstrings and code merged from `brendt/admm-bug`.

Reply:
This should now have been fixed by another merge with a corrected `brendt/admm-bug`.